### DISCUSSION TITLE
cryptography

### DISCUSSION THREAD

**Discussion by Houqua3D** [flet-dev/flet#3164] [Packages] [answered]
`cryptography` is a package which provides cryptographic recipes and primitives to Python developers. Required by `asyncssh` package.
https://pypi.org/project/cryptography/#files

**Comment by FeodorFitsner** [comment, marked as answer]
[`cryptography`](https://pypi.flet.dev/cryptography) and [other packages](https://pypi.flet.dev/) are available in Flet 0.25.0 pre-release and up for both iOS and Android. [Give it a try](https://flet.dev/blog/flet-new-packaging-pre-release) and let us know how that works.
